prow/gerrit/client/client.go

```diff
@@ -246,9 +261,15 @@ func (c *Client) Account(instance string) *gerrit.AccountInfo {
 
 // private handler implementation details
 
-func (h *gerritInstanceHandler) queryAllChanges(lastUpdate time.Time, rateLimit int) []gerrit.ChangeInfo {
+func (h *gerritInstanceHandler) queryAllChanges(lastState *map[string]time.Time, rateLimit int) []gerrit.ChangeInfo {
 	result := []gerrit.ChangeInfo{}
+	timeNow := time.Now()
 	for _, project := range h.projects {
+		lastUpdate, ok := (*lastState)[project]
+		if !ok {
+			logrus.Warnf("could not find lastTime for project %q, probably something went wrong with initTracker?", project)
```

Comment:
`logrus.WithField("project", project).Warn("could not find lastTime blah blah")`

Reply:
done